packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js

```diff
@@ -74,7 +74,7 @@ class TableRow extends React.Component {
     <td key='action' className={styles.actions}>
       <IcoContainer
         icons={[
-          { icoType: 'pencil', onClick: () => this.handleClick(this.props.destination) },
+          { icoType: 'pencil', onClick: this.handleClick },
```

Comment:
`handleClick` defined on line 69 is not accepting parameters. Definition and invocation are not the same.

This should be the more "difficult" to review